lib/composite_primary_keys.rb

```diff
@@ -114,4 +116,4 @@
 
 require_relative 'composite_primary_keys/arel/to_sql'
 require_relative 'composite_primary_keys/arel/sqlserver'
-require_relative 'composite_primary_keys/table_metadata'
\ No newline at end of file
+require_relative 'composite_primary_keys/table_metadata'
```

Comment:
This looks like a needless change - since in the end there is no change.

Reply:
This was my editor just adding a new line at the end of the file. Happy to remove it.